apps/ejabberd/src/mongoose_router_global.erl

```diff
@@ -15,12 +15,16 @@
 %% xmpp_router callback
 -export([filter/3, route/3]).
 
-filter(OrigFrom, OrigTo, OrigPacket) ->
+filter(From, To, OrigPacket) ->
+    Acc = mongoose_stanza:from_element(OrigPacket),
+    Acc1 = mongoose_stanza:put(from, From, Acc),
+    Acc2 = mongoose_stanza:put(to, To, Acc1),
+    Acc3 = mongoose_stanza:put(routing_decision, send, Acc2),
     %% Filter globally
-    case ejabberd_hooks:run_fold(filter_packet,
-        {OrigFrom, OrigTo, OrigPacket}, []) of
-        {From, To, Packet} ->
-            {From, To, Packet};
+    Res = ejabberd_hooks:run_fold(stanza, Acc3, []),
```

Comment:
I don't understand why the hook name has changed from `filter_packet` to `stanza`.